src/lib/utils/common/getWeekNumberByMonth.ts

```diff
@@ -0,0 +1,69 @@
+function getWeekNumberByMonth(dateFormat: Date) {
```

Comment:
* 함수명이 모호합니다. 
* weekNumber 라는 표현은 영문법상 어색한것 같습니다. 
* numberOfWeeks를 사용하면 좋을것 같습니다.

```suggestion
function getNumberOfWeeks(date: Date) {
```

* 반환되는 값이 세가지라 함수명 네이밍이 참 힘든것 같습니다. 상기 함수명은 제 추천일뿐 조금 더 고민하시어 개선하면 좋을것 같습니다.
* dateFormat이라는 변수명이 모호한것 같습니다. 그냥 date 혹은 inputDate로 사용하셔도 될것 같습니다.